app/address/[address]/layout.tsx

```diff
@@ -724,6 +726,27 @@ function getCustomLinkedTabs(pubkey: PublicKey, account: Account) {
         tab: programMultisigTab,
     });
 
+    // Add extensions tab for Token Extensions program accounts
+    if (account.owner.toBase58() === TOKEN_2022_PROGRAM_ADDRESS) {
+        const tokenAccount = account.data.parsed; // as TokenProgramData;
+
+        console.log({ tokenAccount });
+
+        const extensionsTab: Tab = {
+            path: 'token-extensions',
+            slug: 'token-extensions',
+            title: 'Extensions',
+        };
+        tabComponents.push({
+            component: (
+                <React.Suspense key={extensionsTab.slug} fallback={<></>}>
+                    <TokenExtensionsLink tab={extensionsTab} address={pubkey.toString()} />
+                </React.Suspense>
```

Comment:
We shouldn't need to wrap this in a Suspense because we always want to show the `Extensions` tab for T22 accounts, whether or not any exist

Reply:
fixed